storage/gcloud/aio/storage/storage.py

```diff
@@ -485,18 +490,58 @@ def _get_stream_len(stream: IO[AnyStr]) -> int:
             stream.seek(current)
 
     @staticmethod
-    def _preprocess_data(data: Any) -> IO[Any]:
+    def _compress_file_in_chunks(input_stream: Union[IO[AnyStr], io.IOBase],
+                                 output_stream: io.IOBase,
```

Comment:
Rather than passing in a mutable object, please create this locally and return it, eg.
```
def _compress_file_in_chunks(input_stream: Union[IO[AnyStr], io.IOBase], chunk_size: int = 8192) -> None:
    output_stream = io.BytesIO()
    # your existing method
    return output_stream
```